src/Components/Carousel.js

```diff
@@ -1,4 +1,5 @@
 import React , {useState, useEffect} from "react";
+import { getTrendingCarousel } from '../services/carousel'

```

Comment:
The service name may be too specific to this particular implementation. I mean, the carousel is a UI component.
You also may want to get the trending results for something else, like a list or whatever.

I'd rename it to something like `services/trending.js` and the function to be named to something like `getTrendings()` or something like that.